src/FxKit.Tests/UnitTests/Option/Option.NaturalTransformationTests.cs

```diff
@@ -25,6 +25,21 @@ public void OkOrElse_ReturnsTheExpectedValue()
             .BeErr("Err");
     }
 
+    [Test]
+    public void OkOr_ReturnsTheExpectedValue()
+    {
+        Option<int> some = Some(123);
```

Comment:
Here's an example of where implicit casts should work 😄 

```suggestion
        Option<int> some = 123;
```

Reply:
Sometimes it works 😅